nifty-ssl/src/main/java/com/facebook/nifty/ssl/OpenSslServerConfiguration.java

```diff
@@ -62,6 +62,9 @@
         public long sessionTimeoutSeconds = 86400;
         public long sessionCacheSize = 0;
         public boolean enableStatefulSessionCache = true;
+        public int maxSslBufferSize = 19267584;
```

Comment:
Will rename to maxSslBufferBytes.

This number is the default for Netty. However, we currently only use one ssl buffer object. If I can find a way to get the number of IO threads in this class, we can initialize the thread local ssl buffer with maxSslBufferSize / # of IO threads. That way, we'll still use the same amount of memory by default.